src/material-experimental/mdc-chips/harness/chip-grid-harness.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness} from '@angular/cdk-experimental/testing';
+import {MatChipRowHarness} from './chip-row-harness';
+import {MatChipInputHarness} from './chip-input';
+
+/**
+ * Harness for interacting with a mat-chip-grid in tests.
+ * @dynamic
+ */
+export class MatChipGridHarness extends ComponentHarness {
+  static hostSelector = 'mat-chip-grid';
+
+  private _rows = this.locatorForAll(MatChipRowHarness);
+  private _input = this.locatorFor(MatChipInputHarness);
+
+  /** Gets promise of the harnesses for the chip rows. */
+  async getRows(): Promise<MatChipRowHarness[]> {
+    return await this._rows();
+  }
+
+  /** Gets promise of the chip input harness. */
+  async getInput(): Promise<MatChipInputHarness|null> {
```

Comment:
What do you think of `getTextInput`? I'm thinking that just "input" by itself might be too ambiguous

Reply:
SGTM